src/commands/mixu.ts

```diff
@@ -1,13 +1,47 @@
 import { Guild } from 'discord.js'
 import { Discord, SimpleCommand, SimpleCommandMessage } from 'discordx'
+import { injectable } from 'tsyringe'
+import { ORM } from '../persistence'
+
+interface BestMixu {
```

Comment:
For compatibility, you should import this type from prisma instead:
`import { BestMixu } from '../../prisma/generated/prisma-client-js'`

Since SQLite does not support arrays of strings for fields, you'll have to change your code to account for that.

Reply:
Oh I didn't realize there was an autogenerated type you could import from prisma, I'll use that, thank you!